saleor/dashboard/sites/forms.py

```diff
@@ -1,12 +1,19 @@
 from django import forms
+from django.contrib.sites.models import Site
 
 from ...site.models import SiteSettings, AuthorizationKey
 
 
+class SiteForm(forms.ModelForm):
+    class Meta:
+        model = Site
+        exclude = []
+
+
 class SiteSettingForm(forms.ModelForm):
     class Meta:
         model = SiteSettings
-        exclude = []
+        exclude = ['site']
```

Comment:
I'm not a big fan of `exclude`. Can we use `fields` instead? It's safer and much more readable.

Reply:
I think it's okay in this case. In this form we want to edit all fields of the SiteSettings model except this one. I think it's a more general discussion about which practice is better- to use `exclude` or `fields`. If we agreed on one particular way of doing this, we should refactor each form in one separate pull request. @bogdal Please open a ticket if you think that this should be discussed.